src/ZfSnapPhpDebugBar/Service/PhpDebugBarFactory.php

```diff
@@ -38,7 +38,7 @@ public function createService(Locator $serviceLocator)
         $debugbar->addCollector(new ConfigCollector($appConfig, 'ApplicationConfig'));
 
         // Db profiler
-        if ($serviceLocator->has($dbServiceName) && isset($config['db']['driver'])) {
```

Comment:
Please revert this one. I need to write some unit test to reproduce this issue and then we can make something with this.